Automattic/wp-calypso
# Jetpack Backup: Improve copy and styles in restore confirmation pages

Resolves https://github.com/Automattic/jetpack-backup-team/issues/426

## Proposed Changes

* Update regular restore confirmation page header to match granular restore styles which is the preferred option.
* Update restore confirmation pages copy to make it more user-friendly
* Replace all references of `Go Back` to sentence case -> `Go back`

### Full restore confirmation page
Here we mostly changed the header styles and some texts.

| Before | After |
|---|---|
| ![CleanShot 2024-04-22 at 11 07 14@2x](https://github.com/Automattic/wp-calypso/assets/1488641/b321214e-ff93-4f5e-9cf3-bc3e4de6c161) | ![CleanShot 2024-04-22 at 11 07 42@2x](https://github.com/Automattic/wp-calypso/assets/1488641/da7b38d0-6115-4220-9f22-7fff22720d9b) |
| ![CleanShot 2024-04-22 at 11 42 17@2x](https://github.com/Automattic/wp-calypso/assets/1488641/1e5b396d-7d81-4a2b-878a-8d7175e0da1c) | ![CleanShot 2024-04-22 at 11 41 54@2x](https://github.com/Automattic/wp-calypso/assets/1488641/bd9e481c-58fa-439b-8550-6a0274301f90) |
| ![CleanShot 2024-04-22 at 11 44 12@2x](https://github.com/Automattic/wp-calypso/assets/1488641/10306240-a05a-48fa-ae97-1bd9de121c84) | ![CleanShot 2024-04-22 at 11 56 37@2x](https://github.com/Automattic/wp-calypso/assets/1488641/df9481f0-4a45-416f-820b-e15441b8e59d) |
| ![CleanShot 2024-04-22 at 17 59 03@2x](https://github.com/Automattic/wp-calypso/assets/1488641/d51fb555-9c70-4384-8ad6-01a6a2bfd013) | ![CleanShot 2024-04-22 at 17 54 34@2x](https://github.com/Automattic/wp-calypso/assets/1488641/8d499ddd-2354-4c96-b9fe-6b64f25f516d) |

### Granular restore confirmation page
In this page, we changed texts as suggested in https://github.com/Automattic/jetpack-backup-team/issues/426#issuecomment-2047855388. The in progress, success and failed remains the same.

| Before | After |
|---|---|
| ![CleanShot 2024-04-22 at 10 01 13@2x](https://github.com/Automattic/wp-calypso/assets/1488641/648953f3-0267-49b0-a882-1f18cba5b60c) | ![CleanShot 2024-04-22 at 10 00 37@2x](https://github.com/Automattic/wp-calypso/assets/1488641/5387b10e-13b8-4cda-999c-e48786ca3f69) |

## Testing Instructions
* Spin up a Calypso or Jetpack Cloud live branch
* Choose a site with a Jetpack VaultPress Backup plan
* Navigate to the Backup page

To validate the regular restore flow:
* On any backup, click on `Restore to this point`.
* Ensure the pages match the styles and wording described above.

To validate the granular restore flow:
* On any backup, click on `Actions (+)` then on `View files`.
* Select some files in the backup browser
* Click on `Restore selected files`
* Ensure the confirmation page match the style and wording described above.

## Pre-merge Checklist
